build.gradle.kts

```diff
@@ -32,20 +32,20 @@ dependencies {
   compatTestImplementation("org.spockframework:spock-core:2.3-groovy-3.0")
 }
 
-tasks.named<Jar>("jar") {
+tasks.jar {
   manifest {
-    attributes.put("Automatic-Module-Name", "org.ajoberstar.stutter")
+    attributes["Automatic-Module-Name"] = "org.ajoberstar.stutter"
   }
 }
 
-tasks.withType<Test>() {
+tasks.test {
```

Comment:
Original code included all the `compatTest...` tasks, not it's just "unit tests".

Reply:
Good catch!